testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinatorTests.cs

```diff
@@ -38,22 +43,50 @@ private static void ExecuteWithArgs(byte[] args)
             TestCoordinator.Instance.WriteTestResultsServerRpc(s_ValueToValidateAgainst);
         }
 
+        [UnityTest]
+        public IEnumerator CheckPreconditions()
```

Comment:
My understanding is this is a sanity check for bokken?
Any reason why this is in TestCoordinatorTests? Those should have their own set of tests.

Reply:
yeah, good point, I'll move them to a new location.